thread-manager/examples/core_contention_basics.rs

```diff
@@ -48,11 +48,12 @@ fn main() -> anyhow::Result<()> {
         println!("Running {exp}");
         let mut conffile = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
         conffile.push(exp);
-        let conffile = std::fs::File::open(conffile)?;
-        let cfg: RuntimeManagerConfig = serde_json::from_reader(conffile)?;
+        let mut buf = String::new();
+        std::fs::File::open(conffile)?.read_to_string(&mut buf)?;
+        let cfg: RuntimeManagerConfig = toml::from_str(&buf)?;
         //println!("Loaded config {}", serde_json::to_string_pretty(&cfg)?);
 
-        let rtm = RuntimeManager::new(cfg).unwrap();
+        let rtm = ThreadManager::new(cfg).unwrap();
```

Comment:
naming: `rtm`, `cfg`,`tok` is uncommon. we typically use the full descriptive names. Lile rtm -> `thread_manager`

Reply:
done